imports/plugins/included/notifications/server/hooks/notification.js

```diff
@@ -0,0 +1,32 @@
+import { Meteor } from "meteor/meteor";
+import { Logger, MethodHooks } from "/server/api";
+import { Accounts } from "/lib/collections";
+
+const getAdminUserId = () => {
+  const admin = Accounts.find().fetch();
+  return admin[0]._id;
+};
+
+const sendNotificationToAdmin = () => {
+  const adminId = getAdminUserId();
+  const type = "forAdmin";
+  const url = "/reaction/dashboard/orders";
```

Comment:
we actually cannot depend on the url being /reaction/ here.  This is the shop name, slugged.  (isn't it)?

```
  const prefix = Reaction.getSlug(Reaction.getShopName().toLowerCase());
  const url = `/${prefix}/dashboard/orders`;
```